sdk/core/amqp/README.md

```diff
@@ -13,9 +13,7 @@ Some of the common functionalities include:
 
 ## Status
```

Comment:
Considering this guideline https://github.com/Azure/azure-sdk/blob/master/docs/README-TEMPLATE.md I don't think we need the `Status` subtitle at all. We can probably just remove it. Also, we should do that for other packages like servicebus too.

Reply:
@scbedd @weshaggard are the guidelines for minimum requirements? Is it ok to have the `Status` ?because i don't see that consistent across all packages

Reply:
The guidelines are the minimal required so you could have other sections like Status but I agree with you that this Status doesn't really add anything useful so I'd remove it.